Source/Mock.cs

```diff
@@ -834,42 +834,22 @@ private static void SetupAllProperties(Mock mock, Stack<Type> mockedTypesStack)
 					   ProxyFactory.Instance.IsMethodVisible(p.GetGetMethod(), out _))
 				.Distinct();
 
-			var setupPropertyMethod = mock.GetType().GetMethods("SetupProperty")
-				.First(m => m.GetParameters().Length == 2);
-			var setupGetMethod = mock.GetType().GetMethods("SetupGet")
-				.First(m => m.GetParameters().Length == 1);
-
 			foreach (var property in properties)
 			{
 				var expression = GetPropertyExpression(mockType, property);
-				object initialValue = GetInitialValue(mock, mockedTypesStack, property);
+				object value = GetInitialValue(mock, mockedTypesStack, property);
```

Comment:
Renamed because this variable no longer just holds the initial value; it acts as the backing field for the set up property. It gets captured by both setups' getter / setter lambdas.